keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -208,6 +208,47 @@ type RequeueState struct {
 }
 ```
 
+Add a new field `evictionStats` to the WorkloadStatus struct to track the number of evictions by type and reason:
+
+```go
+type WorkloadStatus struct {
+	// evictionStats tracks eviction statistics by type and reason.
+    //
+    // +optional
+    // +listType=map
+    // +listMapKey=type
+    // +patchStrategy=merge
+    // +patchMergeKey=type
+    EvictionStats []Eviction `json:"evictionStats,omitempty"`
```

Comment:
```suggestion
    EvictionStates []WorkloadEvictionState `json:"evictionStates,omitempty"`
```

This has the same potential problem as the previous Topology API which means this could potentially occur conflicts. We should use the identity API typed name.